### ISSUE TITLE
FSNotes_3.3.0.zip is version 3.20 / 261 

### ISSUE THREAD

**Issue by tjluoma** [glushchenko/fsnotes#607]
downloaded FSNotes_3.3.0.zip from https://github.com/glushchenko/fsnotes/releases/tag/3.3.0 but when I installed it, this is the actual version information:

```
/Applications/FSNotes.app:
	CFBundleShortVersionString: 3.2.0
	CFBundleVersion: 261
```

**Comment by glushchenko** [comment]
Hi, @tjluoma. Thank you for feedback. Don't worry, this just not updated variable in Info.plist. Fixed in 3.3.1

**Comment by core-code** [comment]
thanks for solving this, i can confirm this is fixed in 3.3.1:

CFBundleShortVersionString:
3.3.1

CFBundleVersion:
268
